### PR TITLE
Delete order level comments

### PR DESCRIPTION
**Pull request by chengchenghuang**
This PR includes:
1. Delete comments attribute from order model across all files 
2. Distribute comments to individual items when an order is created at the first place
3. Add db/migration_order_comments.sql that should be run on existing database to migrate order comments to its belonging items (tested)

### CODE REVIEW

**Comment by armandofox** [comment]
wait, what's the difference between this PR and the one i just merged?

**Comment by chengchenghuang** [comment]
The previous #39 PR includes: 
Feature 1: remove-order from order view  (e.g. https://rocky-ravine-27276.herokuapp.com/orders?customer_id=1) 
Feature 2: apply effect to individual item in orders view  (when an user edits the comment for an item, it automatically applies to all other items belonging to the same order) 
Feature 3: de-duplicate order-level comments from individual item comments (after edit one comment for one item, and we come back to e.g. https://rocky-ravine-27276.herokuapp.com/customers/1, it is already de-duplicated) 

The #39 PR mostly takes care of basic duplication of an order comment to its items' comments and de-duplication of items' comments. 

The current #41 PR includes:
Feature 1: Delete comments attribute from the actual **ORDER MODEL** (e.g. Order.rb) across all files
Feature 2: Distribute comments to individual items when an order is created at the first place so the comment is never saved to Order in the databse, but as a class variable temporarily 
Feature 3: Add db/migration_order_comments.sql that should be run on existing database to migrate order comments to its belonging items (tested), so this will change the old database forever 

To summarize, #41 will do the correct migration by removing comments attribute from ORDER. Hope this clarifies everything. And thank you in advance for code review.

**Comment by armandofox** [comment]
There are way too many problems with this pull request and I'd be surprised if it even passes the test suite:

- there is Ruby code in a Javascript (.js) file.
- the database schema changes needs to be done as a migration, not a .sql file.  This is a "Rails 101" level issue: the way you change a Rails app's schema is using a migration, always.
- there is the introduction of a new class-level variable `@@comments` in the Order model.  this makes no sense from a code perspective, plus class-level variables are dangerous in code that serves multiple Postgres schemas like this app does.  Even if that weren't the case, though, I cannot understand the motivation for using a class-level variable here.

I'm rejecting this PR entirely.  I think it's better if I implement the schema change, but it is worth going over these problems so you all understand what is wrong.  But this PR is definitely not viable.

I hope/assume the version deployed to rocky-ravine-27276 is the version including the changes of PR#39 but NOT including these changes, as I'm planning to have my colleague/customer liaison inspect the PR#39 changes for final sign-off.
